client/utils/use-payment-method-currencies.js

```diff
@@ -212,10 +212,7 @@ const getKlarnaCurrencies = () => {
 		return [ 'EUR', 'SEK', 'PLN', 'CHF', 'CZK', 'DKK', 'GBP', 'NOK' ];
 	}
 
-	// Throw an error if the country is not recognized.
-	throw new Error(
-		`Unable to determine Klarna currencies for: ${ accountCountry }`
-	);
+	return [];
```

Comment:
We should consider adding a `console.error` (disabling the ' no-console` eslint rule) here for posterity. In case a merchant reaches out in the future. I know we should avoid it, but if a merchant falls into the default return, we will have no clue.